superdesk/media/media_editor.py

```diff
@@ -72,52 +71,62 @@ def transform(self, im, operation, param):
             return ImageEnhance.Contrast(im).enhance(float(param))
         elif operation == 'grayscale':
             return im.convert('L')
+        elif operation == 'saturation':
+            return ImageEnhance.Color(im).enhance(float(param))
         logger.warning('unhandled operation: {operation} {param}'.format(
             operation=operation,
             param=param))
         return im
 
     def create(self, docs):
```

Comment:
I this this could be put into a few methods, it's kind of hard to follow what it does

Reply:
for now it's only calling one method by operation, I could put in separate methods, but that would be on line each time. Do you think it's would really improve readability ?